### PR TITLE
docs(website): add docs for the newly added create-graphback cli

### PR DESCRIPTION
**Pull request by machi1990**

### CODE REVIEW

**Comment by machi1990** [comment]
These are mainly docs changes to accompany https://github.com/aerogear/graphback/pull/1717. 

@wtrocki any chance for a quick review?

**Comment by wtrocki** [comment]
I though we just use existing chapter and we do not need dedicated create-graphback doc

**Comment by wtrocki** [comment]
How this renders in sidebar

**Comment by machi1990** [comment]
Screenshot from 2020-07-15 10-43-24

This is how it looks like.

The getting started section has also been updated to reflect the usage of the CLI.

File: docs/cli/create-graphback.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+:::note
+This part can be skipped if you do not plan to install the CLI globally. 
+:::
+
+If you prefer to install the cli globally, you can do so using your favourite package manager:
+
+With Yarn:
+```bash
+yarn global add create-graphback
```
Comment (wtrocki) [inline docs/cli/create-graphback.md:20]: npx preferred for both

File: docs/cli/create-graphback.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+sidebar_label: Create Graphback
+---
+
+[Create Graphback](https://www.npmjs.com/package/create-graphback) is a command-line utility for creating Graphback applications.
+
+
+## Installation
+
+:::note
+This part can be skipped if you do not plan to install the CLI globally.
```
Comment (wtrocki) [inline docs/cli/create-graphback.md:13]: I would skip that part completely. Things like create react app etc. should be never installed globally as they outdated fast

**Comment by wtrocki** [comment]
All good! Thank you! Our docs looks amazing BTW

**Comment by wtrocki** [comment]
Good to merge!

**Pull request merged**